### ISSUE TITLE
cc2531 lidle led christmas  fairy lights Zigbee Melinera smart LED string lights

### ISSUE THREAD

**Issue by elzershark** [ioBroker/ioBroker.zigbee#874]
cc2531
[Tannenbaum](https://www.lidl.de/de/melinera-lichterkette-zigbee-smart-home/p360021)

```
zigbee.0 | 2020-12-03 23:38:31.618 | info | (1609) Terminated (NO_ERROR): Without reason
-- | -- | -- | --
zigbee.0 | 2020-12-03 23:38:31.616 | info | (1609) terminating
zigbee.0 | 2020-12-03 23:38:31.405 | error | (1609) TypeError: Cannot read property 'id' of undefined at StatesController.syncDevStates (/opt/iobroker/node_modules/iobroker.zigbee/lib/statescontroller.js:282:27) at processTicksAndRejecti
zigbee.0 | 2020-12-03 23:38:31.405 | error | (1609) unhandled promise rejection: Cannot read property 'id' of undefined
zigbee.0 | 2020-12-03 23:38:31.403 | error | Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch().
zigbee.0 | 2020-12-03 23:38:31.361 | error | (1609) TypeError: Cannot read property 'prop' of undefined at /opt/iobroker/node_modules/iobroker.zigbee/lib/statescontroller.js:311:85 at Array.filter (<anonymous>) at StatesController.pu
zigbee.0 | 2020-12-03 23:38:31.360 | error | (1609) unhandled promise rejection: Cannot read property 'prop' of undefined
zigbee.0 | 2020-12-03 23:38:31.359 | error | Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch().
zigbee.0 | 2020-12-03 23:38:31.343 | info | (1609) Zigbee: disabling joining new devices.
zigbee.0 | 2020-12-03 23:38:31.340 | info | (1609) cleaned everything up...
zigbee.0 | 2020-12-03 23:38:31.338 | error | (1609) TypeError: Cannot read property 'prop' of undefined at /opt/iobroker/node_modules/iobroker.zigbee/lib/statescontroller.js:311:85 at Array.filter (<anonymous>) at StatesController.pu
zigbee.0 | 2020-12-03 23:38:31.337 | error | (1609) unhandled promise rejection: Cannot read property 'prop' of undefined
zigbee.0 | 2020-12-03 23:38:31.336 | error | Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch().
zigbee.0 | 2020-12-03 23:38:31.329 | info | (1609) Zigbee started
zigbee.0 | 2020-12-03 23:38:31.327 | info | (1609) 0x588e81fffedb7ebb (addr 41658): HG06467 - Lidl Melinera smart LED string lights (Router)
zigbee.0 | 2020-12-03 23:38:31.320 | info | (1609) Currently 1 devices are joined:
zigbee.0 | 2020-12-03 23:38:31.307 | info | (1609) --> transmitPower : normal
zigbee.0 | 2020-12-03 23:38:31.294 | info | (1609) Coordinator firmware version: {"type":"zStack12","meta":{"transportrev":2,"product":0,"majorrel":2,"minorrel":6,"maintrel":3,"revision":20190608}}
zigbee.0 | 2020-12-03 23:38:29.291 | info | (1609) Installed Version: https://github.com/ioBroker/ioBroker.zigbee/commit/78451a786cbf47a73c9164676ebddaa00135554e
zigbee.0 | 2020-12-03 23:38:29.128 | info | (1609) Starting Zigbee npm ...
zigbee.0 | 2020-12-03 23:38:29.024 | info | (1609) starting. Version 1.3.9-dev in /opt/iobroker/node_modules/iobroker.zigbee, node: v12.20.0, js-controller: 3.1.6
```

**Comment by kirovilya** [comment]
try to update from github and test

**Comment by elzershark** [comment]
Funktioniert Teilweise
works partially

level.color.hue = 
1 = Red 
127 = Green 
255 = blue
hm...

**Comment by kirovilya** [comment]
OK, I'll change it to the classic RGB state, later

**Comment by elzershark** [comment]
it is rgb w-ww
warm white
cold white
rgb

**Comment by DcFamas** [comment]
would be great that is also possible to control the effects shown under "[Trigger effects](https://github.com/Koenkk/zigbee2mqtt.io/blob/master/docs/devices/HG06467.md)"

Thank you! :)

**Comment by kirovilya** [comment]
@DcFamas there is no expose for *effects* :( https://github.com/Koenkk/zigbee-herdsman-converters/blob/master/devices.js#L15386

**Comment by elzershark** [comment]
look here.. https://github.com/Koenkk/zigbee-herdsman-converters/pull/1799#discussion_r530588329

**Comment by Da-ThomasS** [comment]
For me the Colors don‘t work, they are just wrong. Also is there a way to make field for a „zigbee2mqtt/FRIENDLY_NAME/set“ Command, where I can put the json effect string like mentioned here: https://www.zigbee2mqtt.io/devices/HG06467.html ? That would be awesome!

**Comment by elzershark** [comment]
ich weiß nicht warum das nicht integriert wird. Das ist schon lange drin.
https://github.com/Koenkk/zigbee-herdsman-converters/blob/master/converters/common.js

Aber hier wird gesagt, es wäre nicht drin. Sehr, sehr  schade

**Comment by o0shojo0o** [comment]
@elzershark  ...... 
https://github.com/ioBroker/ioBroker.zigbee/pull/889

**Comment by MichaelKorn** [comment]
THX. The control of the device works fine for me.
Unfortunately, I'm a little disappointed with the fairy lights. If you use the effects with your own colors, you may find that occasionally the wrong colors appear. This happens especially at the end of the fairy lights.

**Comment by Da-ThomasS** [comment]
How do you set effects with own colors? Can't get it work.

**Comment by o0shojo0o** [comment]
If I am not mistaken, the brightness can only be used for the warm white area.

**Comment by elzershark** [comment]
Klappt alles.. b.z.w.

zigbee.0.588e81fffedb7ebb.effect_json = 

{"speed":0,"effect":"carnaval","colors":[{"r":0,"g":0,"b":255},{"r":0,"g":255,"b":255},{"r":255,"g":255,"b":255},{"r":0,"g":0,"b":255},{"r":0,"g":255,"b":255},{"r":255,"g":255,"b":255}]}


```
zigbee.0 | 2020-12-10 22:23:47.369 | warn | (16696) This value will not be set in future versions. Please report this to the developer.
-- | -- | -- | --
zigbee.0 | 2020-12-10 22:23:47.368 | warn | (16696) State value to set is invalid for zigbee.0.588e81fffedb7ebb.effect_colors: The state contains the forbidden properties effect, speed, colors!
zigbee.0 | 2020-12-10 22:23:47.353 | warn | (16696) This value will not be set in future versions. Please report this to the developer.
zigbee.0 | 2020-12-10 22:23:47.353 | warn | (16696) State value to set is invalid for zigbee.0.588e81fffedb7ebb.effect_type: The state contains the forbidden properties effect, speed, colors!
zigbee.0 | 2020-12-10 22:23:47.349 | warn | (16696) This value will not be set in future versions. Please report this to the developer.
zigbee.0 | 2020-12-10 22:23:47.348 | warn | (16696) State value to set is invalid for zigbee.0.588e81fffedb7ebb.effect_speed: The state contains the forbidden properties effect, speed, colors!
zigbee.0 | 2020-12-10 22:23:47.346 | warn | (16696) This value will not be set in future versions. Please report this to the developer.
zigbee.0 | 2020-12-10 22:23:47.345 | warn | (16696) State value to set is invalid for zigbee.0.588e81fffedb7ebb.effect_json: The state contains the forbidden properties effect, speed, colors!
  |   | debug | at async /opt/iobroker/node_modules/iobroker.zigbee/main.js:395:32
  |   | debug | at Object.convertSet (/opt/iobroker/node_modules/zigbee-herdsman-converters/converters/toZigbee.js:3917:42)
  |   | debug | at Object.convertStringToHexArray (/opt/iobroker/node_modules/zigbee-herdsman-converters/converters/utils.js:252:31)
```

**Comment by asgothian** [comment]
closed - Unit is working in 1.4.1. Warning messages are caused by zigbee-herdsman-converters, but warned properties are required.
